### PR TITLE
Created medium and medium_master test suites

### PR DESCRIPTION
**Pull request by tplavcic**
Notes:
1. mongos_test is added although I don't think it runs anything but we can check
2. !concurrency_simultaneous is I think ignored I can check later what's with that

### CODE REVIEW

File: suite_sets/resmoke_psmdb_3.2_medium.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+core_auth,default
+core_small_oplog,wiredTiger,PerconaFT,rocksdb
+core_small_oplog_rs,wiredTiger,PerconaFT,rocksdb
+dbtest,mmapv1,wiredTiger,PerconaFT,rocksdb
+disk,default
+dur_jscore_passthrough,default
+durability,default
+failpoints,default
+failpoints_auth,default
+jstestfuzz,mmapv1,wiredTiger,PerconaFT,rocksdb
```
Comment (denis-protyvenskyi) [inline suite_sets/resmoke_psmdb_3.2_medium.txt]: Maybe also comment these out as we don't run them?

Comment (denis-protyvenskyi) [inline suite_sets/resmoke_psmdb_3.2_medium.txt]: and remove `mmapv1` of course

File: suite_sets/resmoke_psmdb_3.2_medium.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+audit,wiredTiger,PerconaFT,rocksdb
+auth,default
+auth_audit,default
+backup,wiredTiger,rocksdb
+bulk_gle_passthrough,wiredTiger,PerconaFT,rocksdb
+concurrency,wiredTiger,rocksdb
+concurrency_replication,wiredTiger,PerconaFT,rocksdb
+concurrency_sharded,wiredTiger,rocksdb
+concurrency_sharded_sccc,default
+!concurrency_simultaneous --executor=concurrency jstests/concurrency/fsm_all_simultaneous.js,mmapv1,wiredTiger,rocksdb
```
Comment (denis-protyvenskyi) [inline suite_sets/resmoke_psmdb_3.2_medium.txt]: remove `mmapv1`

File: suite_sets/resmoke_psmdb_3.2_medium_master.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+core_auth,default
+core_small_oplog,wiredTiger,rocksdb
+core_small_oplog_rs,wiredTiger,rocksdb
+dbtest,mmapv1,wiredTiger,PerconaFT,rocksdb
+disk,default
+dur_jscore_passthrough,default
+durability,default
+failpoints,default
+failpoints_auth,default
+jstestfuzz,mmapv1,wiredTiger,PerconaFT,rocksdb
```
Comment (denis-protyvenskyi) [inline suite_sets/resmoke_psmdb_3.2_medium_master.txt]: `mmapv1`, `PerconaFT`. comment out the whole suite

File: suite_sets/resmoke_psmdb_3.2_medium_master.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+audit,wiredTiger,rocksdb
+auth,default
+auth_audit,default
+backup,wiredTiger,rocksdb
+bulk_gle_passthrough,wiredTiger,rocksdb
+concurrency,wiredTiger,rocksdb
+concurrency_replication,wiredTiger,rocksdb
+concurrency_sharded,wiredTiger,rocksdb
+concurrency_sharded_sccc,default
+!concurrency_simultaneous --executor=concurrency jstests/concurrency/fsm_all_simultaneous.js,mmapv1,wiredTiger,rocksdb
```
Comment (denis-protyvenskyi) [inline suite_sets/resmoke_psmdb_3.2_medium_master.txt]: `mmapv1`

**Comment by tplavcic** [comment]
Should be fixed now

**Pull request merged**